### PR TITLE
Finalize controls

### PR DESCRIPTION
**Pull request by will-hou**
Updates most of the joystick/controller mappings to ones outlined in this [document](https://docs.google.com/spreadsheets/d/118UNu5WV4gCs0rb49JMyyx34j7LCAKGp5G3roTfXYQY/edit#gid=0). Still missing the bed dumping controls. Those will be added as soon as #19 is merged

**This needs to be tested ASAP when we get to Portland**

### CODE REVIEW

File: src/main/kotlin/org/sert2521/bunnybots2019/oi/Oi.kt [hunk 1]
```diff
@@ -24,31 +24,37 @@ val controlModeChooser = SendableChooser<ControlMode>().apply {
...
             tubIntake()
         }
     };
-    { primaryJoystick.getRawButton(Operator.TUBINTAKE_OUT_BUTTON) }.watch() {
+    { secondaryJoystick.getRawButton(Operator.TUBINTAKE_OUT_BUTTON) }.watch {
         whileTrue {
             println("Outtake should be running")
             tubOuttake()
         }
+    };
```
Comment (andrewda) [inline src/main/kotlin/org/sert2521/bunnybots2019/oi/Oi.kt:52]: Are the semicolons here necessary? (And at the end of previous lines)

Comment (will-hou) [reply src/main/kotlin/org/sert2521/bunnybots2019/oi/Oi.kt:52]: I think so, but I'm not sure why. It yells at me if I don't add any semicolons.

**Comment by will-hou** [comment]
@andrewda Shouldn't you be resting? 😛 

Hoping everything went well and that you have a very smooth recovery! Looking forward to seeing you Saturday!

File: src/main/kotlin/org/sert2521/bunnybots2019/oi/Oi.kt (hunk 1 above)
Comment (andrewda) [reply src/main/kotlin/org/sert2521/bunnybots2019/oi/Oi.kt:52]: Huh that's super strange, guess it's just Kotlin oddities!

**Comment by will-hou** [comment]
....Also err don't mind the two abomination that is the two ways we're currently checking for button presses... We'd like to pretty it up but we're hesitant to touch too many things since competition is so soon.

We'll clean it up afterwards, promise 😁

**Comment by andrewda** [comment]
@will-hou hah thank you (I'm just up cause I have to take another round of medicine in a while). As for Saturday, unfortunately there's a good chance I'll be watching the livestream from home 😭 But I'll be sending all the best and cheering you on!!

**Comment by will-hou** [comment]
Awwwww :( Thank you!! ❤️ We'll try our best not to scare you too much. We may be writing our auto on the way there and tuning it between matches...

**Pull request merged**